### DISCUSSION TITLE
Dual Monitor Selection/Setting

### DISCUSSION THREAD

**Discussion by doobes** [elementary/triage#355] [Q&A] [answered]
I've got EOS v7.1 set up with 2 monitors.

The primary monitor is the right side one.

How do I switch that to the left side?

Thanks

Chris

**Comment by ryonakano** [comment, marked as answer]
Try clicking the star button and then click "Apply":

![2023-10-15 21 53 14 のスクリーンショット](https://github.com/elementary/triage/assets/26003928/8f2a6103-9d86-454d-a572-edd0d14e36fa)

**Reply by doobes** [reply]
Thanks
